frontend/packages/dev-console/integration-tests/features/pipelines/pipelines-Created_From_Add_Options.feature

```diff
@@ -2,32 +2,32 @@ Feature: Create Pipeline from Add Options
     As a user I want to view pipeline, create, edit and delete the pipeline
 
 Background:
-    Given user logged into the openshift application
-    And openshift cluster is installed with pipeline operator
-    And user is at the project namespace "AUT_MB_Demo" in dev perspecitve
+    Given openshift cluster is installed with pipeline operator
+    And user has selected namespace "aut-pipelines-add-options"
+    And user is at Add page
 
 
 @regression
 Scenario: From Git Page pipelines section: P-01-TC03
-   Given user is at "Add" page
-   When user clicks From Git card on the +Add page
+   Given user is at Add page
+   When user clicks From Git card on the Add page
    Then user navigates to page with header name Import from git
```

Comment:
```suggestion
   Then user will get redirected to page with header name Import from git
```

Reply:
updated as "user redirects to Import from git page"